export-or-backup-using-dumpling.md

```diff
@@ -10,71 +10,126 @@ category: how-to
 
 如果需要直接备份 SST 文件（KV 对）或者对延迟不敏感的增量备份，请参阅 [BR](/br/backup-and-restore-tool.md)。如果需要实时的增量备份，请参阅 [TiCDC](/ticdc/ticdc-overview.md)。
 
+Dumpling 的更多具体用法可以使用 --help 指令查看，或者查看[中文使用手册](https://github.com/pingcap/dumpling/blob/master/docs/cn/user-guide.md)。
+
 使用 Dumpling 时，需要在已经启动的集群上执行导出命令。本文假设在 `127.0.0.1:4000` 有一个 TiDB 实例，并且这个 TiDB 实例中有无密码的 root 用户。 
 
+## 下载地址
+
+最新版 Dumpling 的下载地址见[下载链接](https://download.pingcap.org/dumpling-nightly-linux-amd64.tar.gz)。
+
 ## 从 TiDB 导出数据
 
-使用如下命令导出数据：
+### 导出到 sql 文件
+
+Dumpling 默认导出数据格式为 sql 文件。也可以通过设置 `--filetype sql` 导出数据到 sql 文件：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   --filetype sql \
   --threads 32 \
   -o /tmp/test \
-  -F $(( 1024 * 1024 * 256 ))
+  -F 256
 ```
 
-上述命令中，`-H`、`-P`、`-u` 分别是地址，端口，用户。如果需要密码验证，可以用 `-p $YOUR_SECRET_PASSWORD` 传给 Dumpling。
+上述命令中，`-h`、`-P`、`-u` 分别是地址，端口，用户。如果需要密码验证，可以用 `-p $YOUR_SECRET_PASSWORD` 传给 Dumpling。
 
-默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。假如导出数据的格式是 CSV（使用 `--filetype csv` 即可导出 CSV 文件），还可以使用 `--sql <SQL>` 导出指定 SQL 选择出来的记录，例如，导出 `test.sbtest1` 中所有 `id < 100` 的记录：
+### 导出到 csv 文件
+
+假如导出数据的格式是 CSV（使用 `--filetype csv` 即可导出 CSV 文件），还可以使用 `--sql <SQL>` 导出指定 SQL 选择出来的记录，例如，导出 `test.sbtest1` 中所有 `id < 100` 的记录：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 ./dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   -o /tmp/test \
   --filetype csv \
-  --sql "select * from `test`.`sbtest1` where id < 100"
+  --sql 'select * from `test`.`sbtest1` where id < 100'
 ```
 
-注意，`--sql` 选项暂时仅仅可用于导出 csv 的场景。但是仍旧可以用 `--where` 来过滤要导出的行，使用以下指令，可以导出所有 `id < 100` 的记录：
-
 > **注意：**
 > 
-> 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
+> 1. `--sql` 选项暂时仅仅可用于导出 csv 的场景。
+>
+> 2. 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
+
+### 筛选导出的数据
+
+#### 使用 `--where` 指令筛选数据
+
+默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 ./dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   -o /tmp/test \
   --where "id < 100"
 ```
 
+上述命令将会导出各个表的 id < 100 的数据。
+
+#### 使用 `--filter` 指令筛选数据
+
+Dumpling 可以通过 `--filter` 指定 table-filter 来筛选特定的库表。table-filter 的语法与 .gitignore 相似，[详细语法参考](https://github.com/pingcap/tidb-tools/blob/master/pkg/table-filter/README.md)。
```

Comment:
@yikeke When you translate this PR, please note that we have a new doc for table filter now (see https://github.com/pingcap/docs-cn/pull/3916 and https://github.com/pingcap/docs/pull/3065), so you can just link to the new doc instead of this README.md
I've updated the Chinese link in this commit https://github.com/pingcap/docs-cn/pull/3916/commits/92a5fd454c31e323bd8e049fffb6406f0ebd3ec7.

Reply:
Sure 👍

Reply:
Updated the link in https://github.com/pingcap/docs/pull/3454/files#diff-cef7ca1bff3a596f6ffa6d6599668e36R87